tests/test_middlewares.py

```diff
@@ -299,6 +321,38 @@ def handler(ex, req, resp, params):
         ]
         self.assertEqual(expectedExecutedMethods, context['executed_methods'])
 
+    def test_order_mw_executed_when_exception_in_rsrc(self):
+        """Test that error in inner middleware leaves"""
+        global context
+
+        class RaiseErrorMiddleware(object):
+
+            def process_resource(self, req, resp, resource):
+                raise Exception("Always fail")
+
+        self.api = falcon.API(middleware=[ExecutedFirstMiddleware(),
+                                          RaiseErrorMiddleware(),
+                                          ExecutedLastMiddleware()])
+
+        def handler(ex, req, resp, params):
+            context['error_handler'] = True
```

Comment:
Missing assertion? `context['error_handler']` is set here, but I can't find anywhere the value is being checked...